pkg/utils/installer/install.go

```diff
@@ -1107,11 +1105,20 @@ func CreateConversionDeployment(ctx context.Context, identity string, directCSIC
 	return nil
 }
 
-func GetConversionCABundle() ([]byte, error) {
-	if len(conversionWebhookCaBundle) == 0 {
-		return []byte{}, ErrEmptyCABundle
+func GetConversionCABundle(ctx context.Context, identity string) ([]byte, error) {
+
+	secret, err := utils.GetKubeClient().CoreV1().Secrets(sanitizeName(identity)).Get(ctx, conversionWebhookCertsSecret, metav1.GetOptions{})
```

Comment:
We can read the CA bytes from the secret itself, instead of reading from global vars